src/main/features/dashboard/views/macro/claimStatus/orders.njk

```diff
@@ -19,10 +19,10 @@
 {% macro reviewReceiver(claim, respondToReviewOrderDeadline) %}
   <div class="status-block">
     {% if claim.reviewOrder.requestedBy === MadeBy.CLAIMANT.value %}
-      <h2 class="heading-small">{{ t('Read {{ claimantName }}’s request', { claimantName: claim.claimData.claimant.name }) }}</h2>
+      <h3 class="heading-small">{{ t('Read {{ claimantName }}’s request', { claimantName: claim.claimData.claimant.name }) }}</h3>
```

Comment:
why the change to h3's? The size and bold should be controlled by the heading-small